.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Welcome to Neat! 👋🎉
+
+Do you need help or have a question? Please check out Neat on [Stack Overflow](https://stackoverflow.com/questions/tagged/neat/). Got an idea for a new feature or found a bug? Please fill out the sections below... thank you 👍
```

Comment:
I was thinking for these templates it might be easier for the contributor if it's all on one line. Not sure tho